waterlog/src/Components/AlertTable.js

```diff
@@ -37,22 +37,29 @@ class AlertTableComponent extends Component {
 		return (
 			<table>
 				<thead>
-					<tr>{this.alertInfo.map((info) => <th key = {info}>{info}</th>)}</tr>
+					<tr>
+						{this.alertInfo.map(info => (
+							<th key={info}>{info}</th>
+						))}
+					</tr>
 				</thead>
 				<tbody>
-					{leaks.map((alert) => (
+					{leaks.map(alert => (
 						<tr
-							key = {alert.id}
-							className = {alert.status === 'Fault' ? 'fault' : ''}
-							onClick = {() => (window.location.href = `alert/segment/${alert.id}`)}
+							key={alert.id}
+							className={alert.status === "Fault" ? "fault" : ""}
```

Comment:
Why are you using strict comparison? Just asking

Reply:
sorry man this issues has been sorted in a pending PR.

Reply:
Ok but I am asking you why?

Reply:
I was not coding this. I only formatted the file. My team member coded and made a separate PR.

Reply:
This was used on a table row to highlight which items has the status of fault

Reply:
So btw, regardless of whether you wrote the code or not, it is all going into the same repository. Each PR is treated as a separate piece of code going into a master branch. Hence, we won't merge a PR in until we're happy with that specific PR. Not saying that this specific thing is an issue, but there's no such thing as 2 parallel PR's. Decide between your team members which PR you would like to go in first and then communicate that to us. We can't approve with a promise of it being fixed later, in the real world that can often not end up happening, then we would have just merged in "bad" code.

Reply:
My bad, I will take into consideration next time.